### ISSUE TITLE
Must not be running Spotify Bluetooth Headset Listener

### ISSUE THREAD

**Issue by robbiet480** [vincent-le-normand/Parrot-Status#13]
I've already fixed the issue, but wanted to note that if you are using [Spotify Bluetooth Headset Listener](https://github.com/jguice/mac-bt-headset-fix) and 0.94 of Parrot-Status, you must quit both and then launch Parrot-Status again, as the two apps conflict. You should also stop launching Spotify Bluetooth Headset Listener at startup. If you don't do this, the two will conflict and cause weird issues like hitting the pause button and the app instantly unpausing, or iTunes randomly opening on button press.

@vincent-le-normand Maybe a popup warning people to quit Spotify Bluetooth Headset Listener would be good?

**Comment by vincent-le-normand** [comment]
Maybe. Parrot Status now includes functionality of Spotify Bluetooth Headset Listener, that's why they conflict (at least when touch control for all caps is enabled). I'll take a look sometime...

**Comment by robbiet480** [comment]
@vincent-le-normand yeah I saw you added it. Super awesome addition! Extremely handy.
